### PR TITLE
js-dom

### PR DESCRIPTION
**Pull request by Barasii**
_Replace task name and links as appropriate and remove this line._

### js-dom

[Demo](https://barasii.github.io/DOM-menu/) |
[Code base](https://github.com/Barasii/DOM-menu)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Barasii/DOM-menu/css/style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+* {
```
Comment (aymkin) [inline submissions/Barasii/DOM-menu/css/style.css]: LGTM, please, have a look at some notes below)

File: submissions/Barasii/DOM-menu/css/style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 1.5;
+    color: #222;
+    background-color: #000;
+	background-size: cover;
```
Comment (aymkin) [inline submissions/Barasii/DOM-menu/css/style.css]: keep the same indentation as above

File: submissions/Barasii/DOM-menu/js/menu.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+					brand.appendChild(logo);	
+				});
+			}
+			else if (target.innerHTML == automobile[0].concern) {
+				deleteImg();
+				insertWelcome();
+			};
+		};
+	});
+});
```
Comment (aymkin) [inline submissions/Barasii/DOM-menu/js/menu.js]: all the rest is fantastic!

File: submissions/Barasii/DOM-menu/js/menu.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+	let menuRow = document.querySelectorAll('.menu-row');
+	Array.from(menuRow).forEach(e => {
+			e.appendChild(link);			
+	});
+});
+ 
+
+let menuLink = document.querySelectorAll('.menu-link');
+const appendFigureImg = Array.from(menuLink).forEach(e => {	
+	e.addEventListener("click", (event) => {
```
Comment (A-Ostrovnyy) [inline submissions/Barasii/DOM-menu/js/menu.js]: Try to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

File: submissions/Barasii/DOM-menu/css/style.css (hunk 2 above)
Comment (Barasii) [reply submissions/Barasii/DOM-menu/css/style.css]: Thank you for your attention and cooperation.

File: submissions/Barasii/DOM-menu/js/menu.js (hunk 3 above)
Comment (Barasii) [reply submissions/Barasii/DOM-menu/js/menu.js]: @aymkin, glad to read it! Could you review my rewrite code?

File: submissions/Barasii/DOM-menu/js/menu.js (hunk 4 above)
Comment (Barasii) [reply submissions/Barasii/DOM-menu/js/menu.js]: @A-Ostrovni, thank you for the review! I liked your links so my new knowledge has let me rewrite this code. Could you review it again?

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**